packages/core/lib/commands/install.js

```diff
@@ -1,35 +1,54 @@
+const { callbackify } = require("util");
+
 const command = {
   command: "install",
   description: "Install a package from the Ethereum Package Registry",
   builder: {},
   help: {
-    usage: "truffle install <package_name>[@<version>]",
+    usage: "truffle install <package-identifier> [--alias]",
```

Comment:
Looks like convention in Truffle is snake case when specifying options of this format. So it should be `package_identifiers`.

Reply:
:+1: I've made this change - but just want to note that `package-identifier` was suggested above by @gnidan.